comment_iowriter.go

```diff
@@ -28,6 +28,8 @@ func (s *RawCommentWriter) Post(_ context.Context, c *Comment) error {
 	return err
 }
 
+func (*RawCommentWriter) ShouldPrependGitRelDir() bool { return false }
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method RawCommentWriter.ShouldPrependGitRelDir should have comment or be unexported
<!-- __reviewdog__:ChBiM2M4ZWE4YTY1MzA2MjkyEg9nb2xpbnQtY2lyY2xlY2k= -->